0002-data-collection-extension/README.md

```diff
@@ -1,96 +1,129 @@
 ---
-name: Data Collection Extension protocol
+name: Data Collection extension protocol
```

Comment:
Title changed to `Data Collection protocol`, text mentions that it's an extension of KP.